docs/source/nlp/tutorials/quickstarts/plot_text_classification.py

```diff
@@ -23,206 +31,267 @@
-#    a long time to calculate. If you have a GPU or a similar device you can use it by installing the appropriate
-#    package versions and passing a ``device`` argument to the ``TextData`` constructor or to the
-#    ``calculate_default_properties`` method.
+# Model Eval #3: Prediction Drift
+# ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 #
+# We note that the Prediction Drift (here in the “Passed” tab) shows no
+# issue. Given that we already know that there is significant Label Drift,
+# this means we have Concept Drift - the labels corresponding to our
+# samples have changed, while the model continues to predict the same
+# labels. You can learn more about the different types of drift and how deepchecks detects them in our
+# :ref:`Drift Guide <drift_user_guide>`.
+
+# %%
+# Running Individual Checks
+# =========================
 #
+# Checks can also be run individually. In this section, we'll show two of the more interesting checks and how you can
+# run them stand-alone.
+#
+# Embeddings Drift
+# ----------------
+#
+# In order to run the embedding drift check you must have text embeddings
+# loaded to both datasets. You can read more about using embeddings in deepchecks NLP in our
+# :ref:`Embeddings Guide <nlp__embeddings_guide>`.
+#
+# In this example, we have the embeddings already
+# pre-calculated:
 
-# Calculate properties
-train.calculate_default_properties()
-test.calculate_default_properties()
+from deepchecks.nlp.datasets.classification.tweet_emotion import load_embeddings
 
-# Run the check
-from deepchecks.nlp.checks import PropertySegmentsPerformance
+train_embeddings, test_embeddings = load_embeddings()
 
-check = PropertySegmentsPerformance(segment_minimum_size_ratio=0.07)
-result = check.run(test, predictions=list(test_preds), probabilities=test_probas)
-result.show()
+train.set_embeddings(train_embeddings)
+test.set_embeddings(test_embeddings)
+
+# %%
+# You can also calculate the embeddings using deepchecks, either using an
+# open-source sentence-transformer or using Open AI’s embedding API.
 
-#%%
-# As we can see, the check found some segments that have lower performance compared to the rest of the dataset. It seems
-# that the model has a harder time predicting the emotions in the "neutral-positive" sentiment range (in our case,
-# between around 0 and 0.45).
+# train.calculate_default_embeddings()
+# test.calculate_default_embeddings()
+
+# %%
 #
-# Data Integrity Checks
-# ---------------------
+
+from deepchecks.nlp.checks import TextEmbeddingsDrift
+
+check = TextEmbeddingsDrift()
+res = check.run(train, test)
+res.show()
+
+# %%
+# Here we can see some distinct segments that distinctly contain more
+# samples from train or more sample for test. For example, if we look at
+# the greenish cluster in the middle we see it’s full of inspirational
+# quotes and saying, and belongs mostly to the test dataset. That is the
```

Comment:
delete the "and saying"
(or if you think it's nicer with it, then add the missing "s" -> sayings)